lib/index.js

```diff
@@ -22,20 +23,17 @@ class Client {
     this.api = api
   }
 
-  async ckanAuthz(url) {
+  async _doBlobAuthz() {
```

Comment:
I think it is an open question whether this is a "private" or public method. Either way could you have a brief docstring.

Reply:
Ok, I will change to use `doBlobAuthz()`. And we can check after